CONTRIBUTING.md

```diff
@@ -134,96 +134,69 @@ yarn test
 
 ## Releases
 
-### Writing a semantic commit message
+### Creating a changeset
 
-Consider whether you are making a visible change to the public Scoobie interface,
-which includes:
+We use [Changesets] to manage package releases.
+You'll see a 🦋 bot gliding around pull requests.
 
-- Top-level exports from [src/index.ts](/src/index.ts)
-- Top-level exports from [webpack/index.js](/webpack/index.js)
-- Bundled configuration files [types.d.ts](/types.d.ts) and [typography.ts](/typography.ts)
-- [npm dependencies](/package.json)
```

Comment:
Should we keep this explanation of the "public interface", or do you reckon it's fairly obvious?